include/pybind11/numpy.h

```diff
@@ -767,6 +769,9 @@ template <typename T, int ExtraFlags = array::forcecast> class array_t : public
     explicit array_t(ShapeContainer shape, const T *ptr = nullptr, handle base = handle())
         : array(std::move(shape), ptr, base) { }
 
+    explicit array_t(size_t count, const T *ptr = nullptr, handle base = handle())
```

Comment:
Is this what you mean?https://github.com/pybind/pybind11/pull/782/commits/a06f2f782d55f6dc2881305ee145c336ce0fec37